feat: configure to get some info command metrics

yaoyinnan opened the pull request:
Specify the index to get info xxx(info data, info stats) by configuring, instead of using info all.

Fixes: #1761

yaoyinnan committed:
feat: configure to get some info command metrics

Specify the index to get info xxx(info data, info stats) by configuring, instead of using info all.

Fixes: #1761

Signed-off-by: yaoyinnan <yaoyinnan@foxmail.com>

AlexStocks commented on tools/pika_exporter/exporter/conf.go:
```diff
@@ -0,0 +1,85 @@
+package exporter
+
+import (
```
怎么也调换下顺序，把 "log" 这种系统库放上面嘛

yaoyinnan committed:
feat: configure to get some info command metrics

Specify the index to get info xxx(info data, info stats) by configuring, instead of using info all.

Fixes: #1761

Signed-off-by: yaoyinnan <yaoyinnan@foxmail.com>
